sequencing/ATACSeq/README.md

```diff
@@ -60,6 +61,26 @@ Important software that will be installed in this environment are: MACS3 (3.0.2)
 
 Other software needed are: BEDTools, Bowtie2, Picard and R. Note R also needs to be installed in the conda environment, as there are some packages that are not available for later versions of R. 
 
+## Sample sheet 
+
+A sample sheet with the metadata about samples is essential to run the ATAC-seq pipeline. This should be located in the META_DATA (0_metadata) folder and should be named as sampleSheet.csv. 
+Samples must be in the same order as in the samples.txt file. An example of information about a sample is shown below.
+
+| sequencingBatch | sampleID | sampleCode | cohort | fraction | individualID | age | gender | clinical | vcfID |
+| --------------- | -------- | ---------- |------- | -------- | ------------ | --- | ------ | -------- | ----- |
+| 11031	| 11031_EX161_NEUN_S1	| EX161_NEUN | BRISTOL_A	| NEUN | 859 | 85	| F |	5	| 201023670019_R08C02_EX161 |
+
+- `sequencingBatch`: sequencing batch number
+- `sampleID`: name of sample in samples.txt and name of fastq file corresponding to the sample. This is usually formed by the sequencingBatchNumber_sampleID_fraction_SNumber.
+- `sampleCode`: ID of sample.
+- `cohort`: brain bank or cohort.
+- `fraction`: cell type
+- `individualID`: ID of the individual to which the sample belongs to. All samples that belong to the same individual must have the same ID.
+- `age`
+- `gender`
+- `clinical`: braak stage
```

Comment:
Does this have to be braak stage? If not, could you put `*e.g.*` before it.

Reply:
yes, it needs to be braak stage. I can change the variable name if it is clearer